### DISCUSSION TITLE
[Feature] Export and import settings and backup albums

### DISCUSSION THREAD

**Discussion by JustToby22** [immich-app/immich#13632] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would love it if there was an option to export to and re-import settings from a config file that can be saved to the phone's storage or SD card.

I'm currently on a phone which holds most of its data on the SD card and said SD card moves to whatever phone I transfer to next. At the same time, I have many different folders marked for backup on said albums, around 20-25. It's quite a hassle to have to remember or read from a screenshot the albums I had and hadn't backed up on my last phone.

I'm thinking an export button which pops up a menu asking you what you want to export: offering backup locations and maybe app settings.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by milosnik22** [comment]
Please add this option
